### PR TITLE
E2E test for Observability upgrade with powerscale

### PR DESCRIPTION
**Pull request by HarishH-DELL**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [X] E2E Test for Obs Upgrade with pscale 
image

- [ ] Test B

### CODE REVIEW

**Review by AkshaySainiDell — CHANGES_REQUESTED** [review]
Looks like some of the changes done in https://github.com/dell/csm-operator/pull/619 are reverted in this PR.
Please review and correct if required.

File: tests/e2e/testfiles/storage_csm_powerscale_observability_val2.yaml [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+      - name: dell-replication-controller-init
+        # image: Defines replication init container image. This shouldn't be changed
+        # Allowed values: string
+        image: dellemc/dell-replication-init:v1.0.0
+
+    # observability: allows to configure observability
+    - name: observability
+      # enabled: Enable/Disable observability
+      enabled: true
+      configVersion: v1.8.0
```
Comment (santhoshatdell) [inline tests/e2e/testfiles/storage_csm_powerscale_observability_val2.yaml:329]: The observability config version in file val2/CR2 is lower than that of in file val1/CR1. Should this be reversed for upgrade scenario?

Comment (abhi16394) [reply tests/e2e/testfiles/storage_csm_powerscale_observability_val2.yaml:329]: yeah i agree with santosh, Please aslo try to upgrade/downgrade to major versions and not minor versions.
Add two E2e test, one for upgrading and the other one for downgrading. You can use the same upgrade method to validate downgrade as well. Please check for downgrade E2E test added in scenarios.yaml

Comment (HarishH-DELL) [reply tests/e2e/testfiles/storage_csm_powerscale_observability_val2.yaml:329]: > The observability config version in file val2/CR2 is lower than that of in file val1/CR1. Should this be reversed for upgrade scenario?

FIxed in New PR https://github.com/dell/csm-operator/pull/637

Comment (HarishH-DELL) [reply tests/e2e/testfiles/storage_csm_powerscale_observability_val2.yaml:329]: > yeah i agree with santosh, Please aslo try to upgrade/downgrade to major versions and not minor versions. Add two E2e test, one for upgrading and the other one for downgrading. You can use the same upgrade method to validate downgrade as well. Please check for downgrade E2E test added in scenarios.yaml

I addressed these comments in new PR https://github.com/dell/csm-operator/pull/637.
Based feature acceptance criteria  i did  upgrade , if you feel need for downgrade please create a story

**Comment by HarishH-DELL** [comment]
New PR for this https://github.com/dell/csm-operator/pull/637

**Pull request closed without merging**